libweb3jsonrpc/Eth.cpp

```diff
@@ -250,13 +250,20 @@ string Eth::eth_sendTransaction(Json::Value const& _json)
 	{
 		TransactionSkeleton t = toTransactionSkeleton(_json);
 		setTransactionDefaults(t);
-		TransactionNotification n = m_ethAccounts.authenticate(t);
-		switch (n.r)
+		pair<TransactionRepercussion, Secret> ar = m_ethAccounts.authenticate(t);
```

Comment:
I wander if that would be possible to sign the transaction here (and return the already signed transaction). It's a bit bad that the RPC wrapper has to deal with private keys.